src/main/riddl/Venues/Reservations/reservationTypeDefinitions.riddl

```diff
@@ -0,0 +1,23 @@
+type Status is any of {Draft, OnHold, Reserved, Active, Past} briefly "Internal status of a Reservation."
+
+type Info is {
```

Comment:
Does Reservation need to have a location: LocationId attribute? A reservation is for an Organization, during TimeRange, at Location...